docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_snowflake_datasource.rst

```diff
@@ -397,10 +404,10 @@ Additional Notes
                 class_name: SimpleSqlalchemyDatasource
                 credentials:
                     drivername: snowflake
-                    username: ${snowflake_username}
```

Comment:
I think we should either keep the variable substitution syntax (${snowflake_username}) or something like "YOUR_SNOWFLAKE_USERNAME_HERE". The brackets make it something in between.

Reply:
Great point. Will update now :)

Reply:
I have a preference for YOUR_SNOWFLAKE_USERNAME_HERE <~

Reply:
I tried the '<user_name>' format, and it made things more confusing, especially with MSSQL connection string

Reply:
now the credentials are represented in `YOUR_SNOWFLAKE_USERNAME`-like format